proj-xcode/PowerAuth2/PowerAuthSDK.m

```diff
@@ -1474,6 +1474,40 @@ - (NSData*) generateInvalidBiometricKey
     }];
 }
 
+- (id<PowerAuthOperationTask>) signJwtWithDevicePrivateKey:(PowerAuthAuthentication*)authentication
+                                                    claims:(NSDictionary<NSString*, NSObject*>*)claims
+                                                  callback:(void(^)(NSString *jwt, NSError *error))callback
+{
+    // Prepare claims data
+    NSData *claimsData = [NSJSONSerialization dataWithJSONObject:claims options:0 error:nil];
+    return [self signDataWithDevicePrivateKey:authentication
+                                         data:claimsData
+                                     callback:^(NSData * _Nullable signature, NSError * _Nullable error) {
+        // Handle error
+        if (error) {
+            callback(nil, error);
+            return;
+        }
+        
+        // Prepare JWT Header
+        NSDictionary *header = [NSDictionary dictionaryWithObjects:@[@"ES256", @"JWT"] forKeys:@[@"alg", @"typ"]];
```

Comment:
I would prefer a more readable form here:
```
@{ @"alg": @"ES256", @"typ": @"JWT" }
```

Reply:
I am an $#&*$, sure this looks way better...